spec/poros/weather_results_spec.rb

```diff
@@ -24,9 +24,10 @@
     expect(@weather.visibility.include?('miles')).to be true
 
     exp_uv_ratings = ['low', 'moderate', 'high', 'very high', 'extreme']
-    act_uv_rating = @weather.uv_index.split('(')[1][0..-2]
+    expect(@weather.uv_index).to eq('n/a') if @weather.uv_index == "n/a"
+    act_uv_rating = @weather.uv_index.split('(')[1][0..-2] if !@weather.uv_index == "n/a"
 
-    expect(exp_uv_ratings.include?(act_uv_rating)).to be true 
+    expect(exp_uv_ratings.include?(act_uv_rating)).to be true if !@weather.uv_index == "n/a"
```

Comment:
if it isnt going to rain there wont be a `rain` value. Note this only works for rain, if it snows I wont know about it